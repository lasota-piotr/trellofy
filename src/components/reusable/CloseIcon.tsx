import React from 'react'

const CloseIcon: React.FC = props => {
  return (
    <svg
      id="Layer_1"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 96 96"
      height="1em"
      {...props}
    >
      <polygon
        fill="#333333"
        points="96,14 82,0 48,34 14,0 0,14 34,48 0,82 14,96 48,62 82,96 96,82 62,48"
      />
    </svg>
  )
}

export default CloseIcon
