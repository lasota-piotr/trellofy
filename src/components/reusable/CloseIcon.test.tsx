import React from 'react'
import { render } from '@testing-library/react'
import CloseIcon from './CloseIcon'

describe('CloseIcon', () => {
  test('should render correctly', () => {
    const {container} = render(<CloseIcon />)

    expect(container.firstChild).toMatchSnapshot()
  });
});
