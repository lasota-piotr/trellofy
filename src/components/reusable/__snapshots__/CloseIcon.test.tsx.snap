// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon should render correctly 1`] = `
<svg
  height="1em"
  id="Layer_1"
  viewBox="0 0 96 96"
  xmlns="http://www.w3.org/2000/svg"
>
  <polygon
    fill="#333333"
    points="96,14 82,0 48,34 14,0 0,14 34,48 0,82 14,96 48,62 82,96 96,82 62,48"
  />
</svg>
`;
